pubmed_dl/main.py

```diff
@@ -1,15 +1,16 @@
 import json
+import typer
 from pubmed_dl.ncbi import get_list_pmid, uids_to_docs
 
-def main():
-    start = input("Enter start date (YYYY/MM/DD): ")
-    end = input("Enter end date (YYYY/MM/DD): ")
-    val = get_list_pmid(start,end)
+def main(start: str, end: str):

```

Comment:
Could we think of more informative names for these variables? E.g. `start_date` and `end_date`. Think of variable names as your first layer of documentation.